server/src/main/java/com/saralein/server/middleware/StaticMiddleware.java

```diff
@@ -0,0 +1,81 @@
+package com.saralein.server.middleware;
+
+import com.saralein.server.FileHelper;
+import com.saralein.server.controller.Controller;
+import com.saralein.server.controller.DirectoryController;
+import com.saralein.server.controller.FileController;
+import com.saralein.server.protocol.Methods;
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Response;
+import com.saralein.server.router.Router;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
```

Comment:
This class feels like a router implementation that also handles requests for static resources rather than middleware.

Reply:
I don't feel one way or another about using it that way, I just think the naming is a little misleading.